app-next/components/TravelCards/TravelCards.jsx

```diff
@@ -1,13 +1,37 @@
-"use client"; // if in Next.js app dir
+"use client"; 
 import { useEffect, useState } from "react";
 import styles from "./TravelCards.module.css";
 import traveldata from "../../mockData/travel-cards.json";
 import Card from "../Card/Card";
 import Link from "next/link";
+import globalcss from '../../app/globals.css';
 
 export default function TravelCards() {
   const [travelCardData, setTravelCardData] = useState([]);
 
+  // const [loading, setLoading] = useState(true);
+  // const [error, setError] = useState(null);
+  
+  // useEffect(() => {
+  //   async function fetchTravelData() {
+  //     try {
+  //       const res = await fetch("/api/travel-cards"); 
+  //       if (!res.ok) {
+  //         throw new Error("Failed to fetch travel data");
```

Comment:
No commented out code in a PR! If it's not used, it should not be part of the codebase